gemfiles/rails_5.0.gemfile

```diff
@@ -15,10 +15,4 @@ platform :ruby do
   gem 'sqlite3', '~> 1.3.0'
 end
 
-platforms :rbx do
```

Comment:
`rbx` has been dropped from our build target since [this commit](https://github.com/zilkey/active_hash/commit/a9f1966f6e04e3122252a04c8995867b0dc8f121#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485) .

Can we also remove `rbx` from `gemfiles` ?